src/main/common/utils/numericUtils.ts

```diff
@@ -1,5 +1,21 @@
+import * as numeral from 'numeral'
 import * as _ from 'lodash'
 
+function convert (strVal: string) {
+  // checks length, followed by a filter by not containing ',' or valid thousand separators
```

Comment:
I don't think this comment is needed, it's clear enough to read the method body. Instead it would be worth to explain why we use both `numeral` and `_` to convert a string to number.